test/cea/cea608_memory_unit.js

```diff
@@ -5,238 +5,301 @@
  */
 
 describe('Cea608Memory', () => {
+  const ceaUtils = shaka.test.CeaUtils;
+
+  const CharSet = shaka.cea.Cea608Memory.CharSet;
+
   /** @type {!shaka.cea.Cea608Memory} */
   let memory;
 
   /** @type {!string} */
   const stream = 'CC1';
 
-  // Returns a closed caption with one nested cue, containing default styles.
-  const createDefaultClosedCaption = (startTime, endTime, payload) => {
-    const topLevelCue = new shaka.text.Cue(startTime, endTime, '');
-    const nestedCue = new shaka.text.Cue(startTime, endTime, payload);
-    nestedCue.color = 'white'; // Default text color.
-    nestedCue.backgroundColor = 'black'; // Default background color.
-    topLevelCue.nestedCues.push(nestedCue);
-
-    return {
-      stream,
-      cue: topLevelCue,
-    };
-  };
-
   beforeEach(() => {
-    memory = new shaka.cea.Cea608Memory(new shaka.cea.AtscDecoder(),
-        0, 0 // F1 + C1 -> CC1
-    );
+    // Create a CC1 Memory: F1 + C1 -> CC1
+    memory = new shaka.cea.Cea608Memory(/* fieldNum= */ 0, /* channelNum= */ 0);
   });
 
   it('adds and emits a series of basic characters from the buffer', () => {
     const text = 'test word';
-    const t1 = 1;
-    const t2 = 2;
+    const startTime = 1;
+    const endTime = 2;
     for (const c of text) {
-      memory.addChar(shaka.cea.Cea608Memory.CharSet.BASIC_NORTH_AMERICAN,
+      memory.addChar(CharSet.BASIC_NORTH_AMERICAN,
           c.charCodeAt(0));
     }
-    memory.forceEmit(t1, t2);
-    const cues = memory.decoder_.getParsedClosedCaptions();
-    const expectedCaptions = [createDefaultClosedCaption(t1, t2, text)];
-    expect(cues).toEqual(expectedCaptions);
+    const caption = memory.forceEmit(startTime, endTime);
+
+    const expectedNestedCue = ceaUtils.createDefaultCue(
+        startTime, endTime, text);
+
+    const expectedNestedCues = [expectedNestedCue];
+
+    const topLevelCue = new shaka.text.Cue(startTime, endTime, '');
+    topLevelCue.nestedCues = expectedNestedCues;
```

Comment:
You could shorten the test somewhat if you did something like:

```js
topLevelCue.nestedCues = [
  ceaUtils.createDefaultCue(startTime, endTime, text),
];
```

instead of using a series of single-use variables.  I think it might even be slightly easier to read.

Same for the other tests below.